client/src/components/Upload/Collection.vue

```diff
@@ -268,8 +268,13 @@ export default {
                     console.debug("Warning, upload response does not contain outputs.", model);
                 }
             });
-            const selection = new Galaxy.currHistoryPanel.collection.constructor(Object.values(models));
-            selection.historyId = Galaxy.currHistoryPanel.model.id;
+            // Build selection object
+            const selection = {
+                models: Object.values(models),
+                historyId: Galaxy.currHistoryPanel.model.id,
+                // Unwatch is unused?
```

Comment:
It seems unused? Do you wanted to follow up here?

Reply:
I dug into it more; it's totally unused in this case and I'll just drop it here.  Thanks for the reminder!